android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt

```diff
@@ -279,7 +279,7 @@ class MainActivity : FlutterActivity() {
                         tmpDir,
                         Aapt.binary(applicationContext).absolutePath,
                         tmpDir.path,
-                        true // TODO: Add option to disable this
+                        false // TODO: Add option to toggle multithreading in UI
```

Comment:
I'm happy to create an Issue on GitHub for this to keep track of this, if at all desired.